core/lib/dependency-graph/network-node.js

```diff
@@ -26,42 +29,49 @@ class NetworkNode extends BaseNode {
    * @return {number}
    */
   get startTime() {
-    return this._record.rendererStartTime * 1000;
+    return this._request.rendererStartTime * 1000;
   }
 
   /**
    * @return {number}
    */
   get endTime() {
-    return this._record.networkEndTime * 1000;
+    return this._request.networkEndTime * 1000;
   }
 
   /**
-   * @return {LH.Artifacts.NetworkRequest}
+   * @return {T}
    */
   get record() {
```

Comment:
I think it's kinda confusing to have `record` and `request` properties. Could we rename `record` to `internalRecord` or something to emphasize which one is the normal use case and which one is for debugging.